src/formatron/integrations/_utils.py

```diff
@@ -11,34 +11,34 @@ def _multiple_replace(replacements: typing.Dict[bytes, bytes], regex: re.Pattern
 Processors = set[typing.Literal["sentencepiece", "<0xHH>", "dot_G"]]
 
 
-def _autodetect_processors(vocab: typing.Dict[str, int]):
-    result = set()
+def _autodetect_processors(vocab: typing.Dict[str, int]) -> list[tuple[str, typing.Callable]]:
```

Comment:
since we are now using callables, I think we do not need to return the str at all?

Reply:
That's right, no need for that, especially now that those update functions are extracted and processor names are documented as part of their signature